src/django_components/management/commands/startcomponent.py

```diff
@@ -7,49 +7,117 @@
 
 
 class Command(BaseCommand):
```

Comment:
Same as mentioned in other comments. I want to auto-generate reference docs also for the management commands. So I've put the docs from README in here.

Plus for commands, I generate the inputs section by letting django / argparse to format the arguments, and then I use that to generate the documentation.